### PR TITLE
Wide-lane ambiguity fixing for multi-constellation, L1/L2 or L1/L5

### PR DESCRIPTION
**Pull request by tao-swift**
This PR implemented wide-lane support for GPS&GAL (potentially BDS-3) using L1/L5 frequencies. Satellites with dual-frequency L1/L5 observations will be attempted to fix the ambiguties.

### CODE REVIEW

File: app/rnx2rtkp/rnx2rtkp.c [hunk 1]
```diff
@@ -155,7 +155,10 @@ int main(int argc, char **argv)
         else if (!strcmp(argv[i],"-b")) prcopt.soltype=1;
         else if (!strcmp(argv[i],"-c")) prcopt.soltype=2;
         else if (!strcmp(argv[i],"-i")) prcopt.modear=2;
-        else if (!strcmp(argv[i],"-h")) prcopt.modear=3;
+        else if (!strcmp(argv[i],"-h")) {
+            prcopt.modear=3;
```
Comment (dgburr) [inline app/rnx2rtkp/rnx2rtkp.c:159]: @tao-swift: this doesn't account for the possibility that the user may specify `-w` before `-h`.  With your approach, `modear` will first be set to 6 (==ARMODE_WL) due to the `-w` flag and will then change to 3 (==ARMODE_FIXHOLD) due to the `-h` flag.  This means that the widelane support will not be used.

Comment (dgburr) [reply app/rnx2rtkp/rnx2rtkp.c:159]: What is wrong with the approach which I proposed?  It correctly handles both the `-w -h` and `-h -w` cases (as well as `-h` separately from `-w`)

Comment (dgburr) [reply app/rnx2rtkp/rnx2rtkp.c:159]: Alternatively you could change the above to something like `if(prcopt.modear!=ARMODE_WL) prcopt.modear=ARMODE_FIXHOLD`

Comment (tao-swift) [reply app/rnx2rtkp/rnx2rtkp.c:159]: yeah, you are right. Feel free to merge yours in. The alternative change that you mentioned ^ also looks good to me.

Probably not a good idea to use this fix and hold option, but we can include it in.

**Pull request merged**